api/envoy/config/filter/http/jwt_authn/v2alpha/config.proto

```diff
@@ -157,6 +157,24 @@ message JwtProvider {
   //
   // If it is not specified, the payload will not be forwarded.
   string forward_payload_header = 8;
+
+  // If non empty, successfully verified JWT payloads will be written to StreamInfo DynamicMetadata
+  // in the format as: *namespace* is the jwt_authn filter name as **envoy.filters.http.jwt_authn**
+  // The value is the *protobuf::Struct*. The value of this field will be the key for its *fields*
```

Comment:
This is a pretty confusing interface. Why not put this under a fixed key like `payload_metadata`?

Reply:
You may have multiple JWT entries in a single request and they go to different scope.